client/blocks/importer/wordpress/upgrade-plan/hooks/use-get-upgrade-plan-site-metrics.tsx

```diff
@@ -1,39 +1,48 @@
 import { useIsEnglishLocale } from '@automattic/i18n-utils';
 import { getQueryArg } from '@wordpress/url';
 import { useUrlBasicMetricsQuery } from 'calypso/data/site-profiler/use-url-basic-metrics-query';
-import { upgradePlanSiteMetricsLcpThreshold } from '../constants';
+import {
+	upgradePlanSiteMetricsLcpThreshold,
+	upgradePlanSiteMetricsFidThreshold,
+} from '../constants';
+
+const calcPercentageDifferenceFromThreshold = ( value: number | undefined, threshold: number ) => {
+	if ( ! value || value <= threshold ) {
+		return 0;
+	}
+
+	return Math.round( 100 * Math.abs( ( value - threshold ) / ( ( value + threshold ) / 2 ) ) );
```

Comment:
Do I understand correctly that this `abs` may create a falsehood? If the original site is 10% faster, the user will be told that WP.com faster, no?

Reply:
Oh yes, in that case we don't need the `abs` since `value` is always larger than the `threshold`.

Reply:
Will do! Thanks! I'll just wait for the result of that other thread to update the calc in a single commit. 😉